src/managed/Microsoft.NET.HostModel/AppHost/AppNameTooLongException.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.NET.HostModel.AppHost
+{
+    /// <summary>
+    /// Given app file name is longer than 1024 bytes
+    /// </summary>
+    public class AppNameTooLongException : AppHostUpdateException
+    {
+        public string LongName;
```

Comment:
nit: maybe just expose {get;}

Reply:
Fixed it in this commit: https://github.com/dotnet/core-setup/pull/6831/commits/ff295f51bdbdaf941a053279ed7bd7f470f5529f